src/common/options.cc

```diff
@@ -4935,6 +4935,22 @@ std::vector<Option> get_global_options() {
     .set_default(60)
     .set_description("log collection list operation if it's slower than this age (seconds)"),
 
+    Option("bluestore_volume_selection_policy", Option::TYPE_STR, Option::LEVEL_DEV)
```

Comment:
Any reason why it is in `LEVEL_DEV`? It seems if we want to use this feature we should change this option to `use_some_extra` and when it is in `LEVEL_DEV` we can't use it in production environment!